mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp

```diff
@@ -254,17 +254,28 @@ USER_DEFINED_WRAPPER(int, Waitany, (int) count,
   int flag = 0;
   bool all_null = true;
   *local_index = MPI_UNDEFINED;
+  int *is_null = (int*) malloc(sizeof(int) * count);
```

Comment:
@karya0 Thanks for the suggestion. I was worried about the compatibility of the compiler because I somehow remembered dynamic array is new to the C standard. Since we are using a C++ compiler for this file anyway, a dynamic array is more reasonable.